mods/d2k/rules/arrakis.yaml

```diff
@@ -1,20 +1,19 @@
 spicebloom:
 	HiddenUnderShroud:
-	RenderBuilding:
-	Building:
-		Footprint: x
-		Dimensions: 1,1
+	RenderSprites:
 	AppearsOnRadar:
+		UseLocation: yes
+	EditorAppearance:
+		RelativeToTopLeft: yes
+	ProximityCaptor:
+		Types: Tree
```

Comment:
Thanks for the help, but I already knew about the style nits :/

On 2015-07-04, at 4:12 AM, Matthias Mailänder <notifications@github.com> wrote:

> In mods/d2k/rules/arrakis.yaml:
> 
> >  	AppearsOnRadar:
> > +		UseLocation: yes
> > +	EditorAppearance:
> > +		RelativeToTopLeft: yes
> > +	ProximityCaptor:
> > +		Types: Tree
> I don't think we add this back again. It was removed on purpose.
> 
> —
> Reply to this email directly or view it on GitHub.
>